modules/system/models/EventLog.php

```diff
@@ -84,6 +84,9 @@ public function getSummaryAttribute()
         if (preg_match("/with message '(.+)' in/", $this->message, $match)) {
             return $match[1];
         }
+        elseif (preg_match("/^(.*|\n)/im", trim($this->message), $match)) {
```

Comment:
@ayumi-cloud it's a really bad idea to have `.*` regexp in here. And your newline part - won't work. Mainly you'd have a carriage return at the beginning. Long story short, is a better regexp for a newline character (sort of): `(\r\n|\r|\n)`

And if you'd like to have `<br>` after a newline character better to find those and insert before.

Details:
- in Unix and all Unix-like systems, \n is the code for end-of-line, \r means nothing special
- as a consequence, in C and most languages that somehow copy it (even remotely), \n is the standard escape sequence for end of line (translated to/from OS-specific sequences as needed)
- in old Mac systems (pre-OS X), \r was the code for end-of-line instead
-  in Windows (and many old OSs), the code for end of line is 2 characters, \r\n, in this order
- as a (surprising;-) consequence (harking back to OSs much older than Windows), \r\n is the standard line-termination for text formats on the Internet
- for electromechanical teletype-like "terminals", \r commands the carriage to go back leftwards until it hits the leftmost stop (a slow operation), \n commands the roller to roll up one line (a much faster operation) -- that's the reason you always have \r before \n, so that the roller can move while the carriage is still going leftwards!-) Wikipedia has a more detailed explanation.
- for character-mode terminals (typically emulating even-older printing ones as above), in raw mode, \r and \n act similarly (except both in terms of the cursor, as there is no carriage or roller;-)

Reply:
@w20k Thanks, I totally forgot about adding support for Mac's. Going to save this comment in my notes, thanks for going the extra mile with all the details, very useful. I remember those days using `\r`

I'm going to add your regex in a test website and run it for a little while to test it and then I will update the pr.

Thanks for your review 👍